Userland/Utilities/lscpu.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, Undefine <undefine@undefine.pl>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "AK/Forward.h"
+#include <AK/Format.h>
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+static void print_cpu_info(JsonObject const& value)
+{
+    outln("CPU {}:", value.get("processor").as_u32());
+    outln("\tVendor ID: {}", value.get("cpuid").as_string());
+    outln("\tModel: {}", value.get("brand").as_string());
+    outln("\tFamily: {}", value.get("family").as_u32());
+    outln("\tModel: {}", value.get("model").as_u32());
+    outln("\tStepping: {}", value.get("stepping").as_u32());
+    outln("\tType: {}", value.get("type").as_u32());
+    out("\tFeatures: ");
+
+    auto& features = value.get("features").as_array();
+
+    for (const JsonValue& feature : features.values())
+        out("{} ", feature.as_string());
+
+    out("\n\n");
```

Comment:
This only was changed from `outln()` x 2 to printing newlines manually. Please implement the suggestion I made :)

Reply:
But what do you mean to put it in the outer function?